token/keycloak/external_token_test.go

```diff
@@ -0,0 +1,93 @@
+package keycloak_test
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/fabric8-services/fabric8-auth/configuration"
+	"github.com/fabric8-services/fabric8-auth/errors"
+	"github.com/fabric8-services/fabric8-auth/resource"
+	testsuite "github.com/fabric8-services/fabric8-auth/test/suite"
+	"github.com/fabric8-services/fabric8-auth/token/keycloak"
+
+	"github.com/fabric8-services/fabric8-auth/token"
+	"github.com/goadesign/goa"
+	_ "github.com/lib/pq"
+	errs "github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type KeycloakExternakTokenTest struct {
+	testsuite.RemoteTestSuite
+	clean                        func()
+	KeycloakExternalTokenService keycloak.KeycloakExternalTokenService
+	accessToken                  *string
+	keycloakExternalTokenURL     *string
+}
+
+func TestRunKeycloakExternakTokenTest(t *testing.T) {
```

Comment:
Let's not add new KC remote tests especially for temporal code.
I'm pretty sure this test will fail from time to time. BTW I think I know why our KC tests which work with test users fail from time to time. When we "login" a test user a new session is created on KC side. But the session lives only in Infinispan cache. So it takes some time for the cache to sync the sessions among the KC cluster. And in our tests we are using the test token right after login. And if we hit another KC pod where session cache has not synced yet then it fails.

Reply:
Understood, safe to remove now - given that I've run the tests a few times locally and the code has been 'verified'. 
0633d70